vendor/bat-native-ads/include/bat/ads/public/interfaces/ads.mojom

```diff
@@ -141,3 +141,28 @@ struct DBCommandResponse {
   DBCommandResult? result;
   Status status;
 };
+
+enum CovariateType {
+  kAdNotificationClicked,
+  kLocaleCountryAtTimeOfServing,
+  kImpressionServedAt,
+  kNumberOfTabsOpenedInPast30Minutes
```

Comment:
Would these eventually be for all add types or just kAdNotifications? If just ad notifications maybe we should make this future proof by prepending `kAdNotification`